python/lib/communication/dmod/communication/update_message.py

```diff
@@ -28,143 +31,66 @@ class type, but note that when messages are serialized, it is converted to the f
     update it conveys.
     """
 
-    event_type: MessageEventType = MessageEventType.INFORMATION_UPDATE
+    event_type: ClassVar[MessageEventType] = MessageEventType.INFORMATION_UPDATE
+
+    object_id: str = Field(description="The identifier for the object being updated, as a string.")
+    object_type: Type[object] = Field(description="The type of object being updated.")
```

Comment:
I am not sure. I want to say for covariance reasons this needs to be `Type[object]`, but ill look into this and revisit this. @aaraney #TODO

Reply:
> `Type[object]` is equivalent to `type` and `typing.Type` since it just declares that `object_type` is just the type of _something_.

`typing.Type[T]` is saying some type that is covariant  in `T` (i.e subclasses), _not_ an instance of `T`. So this is just saying any python type, _not_ any python instance. I _think_ this is equivalent to `typing.Type[typing.Any]`.